mapic/misttriggers/stream_buffer.go

```diff
@@ -82,12 +84,16 @@ func TriggerStreamBuffer(cli *config.Cli, req *http.Request, lines []string) err
 }
 
 type StreamHealthPayload struct {
-	StreamName string                  `json:"stream_name"`
-	SessionID  string                  `json:"session_id"`
-	IsActive   bool                    `json:"is_active"`
-	IsHealthy  bool                    `json:"is_healthy"`
-	Tracks     map[string]TrackDetails `json:"tracks,omitempty"`
-	Issues     string                  `json:"issues,omitempty"`
+	StreamName string `json:"stream_name"`
+	SessionID  string `json:"session_id"`
+	IsActive   bool   `json:"is_active"`
+
+	IsHealthy   bool   `json:"is_healthy"`
+	Issues      string `json:"issues,omitempty"`
+	HumanIssues string `json:"human_issues,omitempty"`
+
+	Tracks map[string]TrackDetails `json:"tracks,omitempty"`
```

Comment:
Not blocking release, but I kind of think we shouldn't in general use `json.Unmarshal` with stuff coming out of Mist; I don't like the idea of them adding a field and our code breaks. Unless there's a way to signify `ignoreExtra` to the JSON unmarhsaller

Reply:
JSON unmarshalling does ignore extra fields! It just doesn't ignore type mismatches, which is why the previous code was breaking (trying to parse a number as an object). Jaron guaranteed that the only "object fields" that we'll have in this payload will be tracks, and that they make sure not to break compatibility with existing clients, so I believe we are safe now!

Reply:
ohhhh cool